tools/wait_for_cluster_info.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import waiting
+
+import utils
+import deployment_options
+from retry import retry
+
+TIMEOUT = 60 * 30
+SLEEP = 10
+
+
+def handle_arguments():
+    return deployment_options.load_deployment_options()
+
+
+@retry(exceptions=RuntimeError, tries=2, delay=3)
+def is_cluster_info_ready(namespace, kubectl_cmd) -> bool:
+    print(f"DEBUG - is cluster-info ready (kubectl_cmd: {kubectl_cmd} namespace: {namespace} )")
+    cmd = f"{kubectl_cmd} cluster-info"
+    try:
+        print(cmd)
+        utils.check_output(cmd)
+    except RuntimeError:
+        print("DEBUG - cluster is not ready yet.")
+        return False
+    return True
+
+
+def main():
+    deploy_options = handle_arguments()
+    if not deploy_options.apply_manifest:
+        return
+
+    waiting.wait(
+        lambda: is_cluster_info_ready(
+            namespace=deploy_options.namespace,
+            kubectl_cmd=utils.get_kubectl_command(deploy_options.target, deploy_options.namespace)
```

Comment:
(`get_kubectl_command` is OK with `namespace=None`, and it's the default value, so it's not really needed here either)

Reply:
what about custom namespaces? as we use in e2e jobs?

Reply:
cluster-info doesn't need any namespace

Reply:
This is how we used it, so I assumed as such. But looks like indeed this is the case. I'll remove it.

Reply:
Done